plugins/com.google.cloud.tools.eclipse.appengine.newproject.maven/src/com/google/cloud/tools/eclipse/appengine/newproject/maven/MavenAppEngineStandardWizardPage.java

```diff
@@ -364,14 +364,16 @@ public void verifyText(VerifyEvent event) {
       if (MavenCoordinatesValidator.validateGroupId(newGroupId.trim())) {
         String newSuggestion = suggestPackageName(newGroupId);
         updatePackageField(newSuggestion);
-        previousSuggestion = newSuggestion;
+      } else if (newGroupId.trim().isEmpty()) {
+        updatePackageField("");
       }
     }
 
     private void updatePackageField(String newSuggestion) {
       if (getPackageName().isEmpty() || getPackageName().equals(previousSuggestion)) {
         javaPackageField.setText(newSuggestion);
       }
+      previousSuggestion = newSuggestion;
```

Comment:
This will override the package field when explicitly set. Steps to repeat:
  1. Open the new maven wizard
  2. Put "foo" in the Java package field
  3. Start writing "footer" in the group field.  See the Java package field change.